gems/sorbet-runtime/lib/types/_types.rb

```diff
@@ -239,7 +239,7 @@ def self.absurd(value)
     begin
       raise TypeError.new(msg)
     rescue TypeError => e # raise into rescue to ensure e.backtrace is populated
-      T::Configuration.inline_type_error_handler(e)
+      T::Configuration.inline_type_error_handler(e, kind: 'T.absurd', value: value, type: nil)
```

Comment:
these kinds of things will cause warnings in future ruby versions. can you wrap the keyword args in `{ ... }`? (or alternatively, declare `inline_type_error_handler` as taking keyword `**opts`, but I prefer making there be a hash literal at the call site)

Reply:
Update to use a hash 👍